x/poe/contract/tgrade_valset_integration_test.go

```diff
@@ -280,3 +282,34 @@ func TestJailUnjail(t *testing.T) {
 	}
 
 }
+
+func TestToValidator(t *testing.T) {
```

Comment:
🚀  very nice test. Can you move it to a new `tgrade_valset_test.go` file?
"_integration_test.go" call a real contract instance (by convention)